examples/2d/drag_scale_camera.rs

```diff
@@ -0,0 +1,73 @@
+//! How to change the transform and scale of the camera. Affect camera via dragging and scrolling
+
+use bevy::{input::mouse::MouseWheel, prelude::*, sprite::MaterialMesh2dBundle};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(camera_scale)
+        .add_system(camer_drag)
+        .insert_resource(ClearColor(Color::RED))
+        .run();
+}
+
+fn camer_drag(
+    mouse_button_input: Res<Input<MouseButton>>,
+    mut query: Query<(&mut Transform, &mut OrthographicProjection)>,
+
+    winfo: Res<Windows>,
```

Comment:
```suggestion
    windows: Res<Windows>,
```
We don't need to save characters here.